### DISCUSSION TITLE
Auto Update on webpage (HTTP)

### DISCUSSION THREAD

**Discussion by mglazzari-qinmotion** [cesanta/mongoose#2868] [Ask questions, get answers] [answered]
I am building a dashboard with the Wizard, and set the Heartbeat to 5. The description of the Heartbeat field in the Wizard reads "UI auto-update interval in seconds". I was expecting this to update all the variables displayed on the dashboard every 5 seconds, but the only thing it updates is the red/green dot that shows the connection status.

Is there any way to set up the Wizard (or the mongoose_glue file) to get the values in the dashboard to update periodically?

**Comment by scaprile** [comment, marked as answer]
https://discord.gg/KfR8E6wSds
![Screenshot from 2024-08-22 13-55-48](https://github.com/user-attachments/assets/af0eba03-aeea-456c-be9a-d1d485fc22bf)
https://github.com/cesanta/mongoose-wizard

**Comment by mglazzari-qinmotion** [comment]
OK, I think you mean to say that it relates to a different project (the wizard) and that I should check the Wizard documentation or post to the Discord channel.
I looked up in the closed issues in the Wizard project in Github and there is a function that the user can call to update the variables on the next heartbeat:
```c
void glue_update_state(void); 
```
Tested and it works, thanks
